spec/index.html

```diff
@@ -365,6 +380,18 @@ <h3>Terms defined by this specification</h3>
       <dd>A tuple composed of <a>subject</a>, <a>predicate</a>, <a>object</a>, and <a>graph name</a>.
         This is a generalization of an <a>RDF triple</a> along with a <a>graph name</a>.
       </dd>
+      <dt><dfn class="export">canonicalization function</dfn></dt>
+      <dd>A <a>canonicalization function</a> maps <a>RDF datasets</a>
+        into <a data-cite="RDF11-CONCEPTS#dfn-isomorphic">isomorphic datasets</a>.
+        Two datasets produce the same canonical result if and only if they are <a data-cite="RDF11-CONCEPTS#dfn-isomorphic">isomorphic</a>.
+        The <a>RDFC-1.0</a> algorithm implements a <a>canonicalization function</a>,
+        although some datasets may be constructed to prevent this algorithm from
+        terminating (see <a href="#dataset-poisoning" class="sectionRef"></a>).
+        In this case the algorithm can be considered to be
+        a <dfn class="export">canonicalization partial function</dfn>.
```

Comment:
Could the order `canonicalization partial` be flipped? `partial canonicalization function` makes more sense to me, and I'd strongly prefer it unless there's a strong argument for the existing `canonicalization partial function`.
```suggestion
        a <dfn class="export">partial canonicalization function</dfn>.
```

Reply:
We might need a term to classify poison datasets.